draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -893,19 +893,32 @@ a different salt.
 
 ## Choice of a Hash Algorithm
 
-For the security of this scheme, the hash algorithm is required to be preimage resistant and second-preimage
-resistant, i.e., it is infeasible to calculate the salt and claim value that result in
-a particular digest, and, for any salt and claim value pair, it is infeasible to find a different salt and claim value pair that
-result in the same digest, respectively.
-
-Hash algorithms that do not meet the aforementioned requirements MUST NOT be used.
-Inclusion in the "Named Information Hash Algorithm" registry [@IANA.Hash.Algorithms]
+To ensure privacy of claims that are not being selectively disclosed in a given presentation,
+the hash function MUST ensure that it is infeasible to calculate the salt and claim name-value pair
+(or any portion thereof) that results in a particular digest. This implies the hash function MUST
```

Comment:
`claim name-value pair` is somewhat confusing and is not used elsewhere in the spec for this purpose.

```suggestion
the hash function MUST ensure that it is infeasible to calculate one or more of the three elements
(salt, claim name, claim value) that results in a particular digest. This implies the hash function MUST
```

Reply:
```suggestion
the hash function MUST ensure that it is infeasible to calculate any portion of the three elements
(salt, claim name, claim value) from a particular digest. This implies the hash function MUST
```